### PR TITLE
#159691609 Track Comment Edit History 

### PR DESCRIPTION
**Pull request by johnmusiu**
#### What does this PR do?
Implements track comment edit history

#### Description of tasks to be completed
List of implemented features:
1. User can edit their comment 
2. A comment should indicate it was edited
3. User should see their comment edit history

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans/tree/ft-track-comment-edit-history-159691609
```
```sh
$ cd ah-titans
```
> Create a virtual environment and activate it.

```sh
$ virtualenv -p python3 venv
```
```sh
$ source venv/bin/activate
```

> Add **.env** file and copy **example.env** content.

``` cp example.env .env```

> Add your local configuration settings to your **.env** file

```sh
$ source .env
```

```sh
$ pip install -r requirements.txt
```

```sh
$ python manage.py makemigrations
```

```sh
$ python manage.py migrate
```

```sh
$ python manage.py runserver
```

Now you can test the various endpoint on the postman.

#### Any background context you want to provide?
None.

#### What are the relevant Pivotal Tracker Stories?
[#159691609](https://www.pivotaltracker.com/story/show/159691609)

image

> Get edit history
image

#### Relevant templates
image

image

### CODE REVIEW

**Review by dcDalin — APPROVED** [review]

**Review by janetnim — APPROVED** [review]

**Comment by janetnim** [comment]
I think you should give a screenshot of the template of how the edit history of a comment will look like

**Review by janetnim — CHANGES_REQUESTED** [review]
I think you should give a screenshot of the template of how the edit history of a comment will look like

**Comment by chepmanu** [comment]
Looks good to me.

**Review by migot01 — APPROVED** [review]

File: authors/tests/articles/test_comments.py [hunk 1]
```diff
@@ -246,8 +245,36 @@ def test_cannot_delete_inexisting_comment(self):
         token = self.login_verified_user(self.test_user)
         self.create_article(token, self.article)
         self.create_comment(token, 'tests', self.comment)
-        response = self.delete_comment(token, 'test',self.comment,'2')
+        response = self.delete_comment(token, 'test', self.comment, '2')
         self.assertIn("A comment with this ID does not exist.",
                       response.content.decode())
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
-
```
Comment (migot01) [inline authors/tests/articles/test_comments.py:253]: Nice work , though you should remove the extra line here

**Review by migot01 — CHANGES_REQUESTED** [review]

**Review by migot01 — APPROVED** [review]

**Review by janetnim — APPROVED** [review]

**Pull request merged**